core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/ApplicationSummaryInfo.scala

```diff
@@ -136,4 +146,18 @@ class SingleAppSummaryInfoProvider(val app: ApplicationSummaryInfo)
   override def getRapidsJars: Seq[String] = {
     app.rapidsJar.map(_.jar).seq
   }
+
+  override def isParquetOrOrc: Boolean = {
+    app.dsInfo.map(_.format).distinct.exists(str => str.contains("Parquet")) ||
+       app.dsInfo.map(_.format).distinct.exists(str => str.contains("ORC"))
```

Comment:
Would be nice to not scan this more than once.  Also, I'm not sure it's worth doing the distinct pass before doing the exists check, since with just an exists check we can early out, but not if we have to do a full distinct pass first.
```suggestion
    app.dsInfo.exists { info =>
      info.format.contains("Parquet") || info.format.contains("ORC")
    }
```

Reply:
Good feedback.  Changed in latest commit.  And removed ORC for now.